platforms/bukkit/build.gradle.kts

```diff
@@ -3,7 +3,9 @@ plugins {
 }
 
 repositories {
-    maven("https://s01.oss.sonatype.org/content/repositories/snapshots/")
+    maven("https://s01.oss.sonatype.org/content/repositories/snapshots/") {
+        name = "sonatype"
```

Comment:
very minor nitpick, not really an issue: "Sonatype" should be capitalized to fit with the implicit precedent set by the other repo names

Reply:
Will fix

Reply:
also same for "devmart-other", though unsure how that should be capitalized

though, you don't have to fix it, it's not a big deal